integrations/extensions/starter-kits/google-custom-search/README.md

```diff
@@ -46,86 +59,118 @@ If you want to add this starter kit to an _existing_ assistant, you cannot use t
-- Optional: In "Assistant says", put `Searching for: ${query_text}`.
-- In "And then", select "Use an extension", and select the search endpoint and set the `query` parameter to the `query_text` session variable and the `cx` parameter to `cx` session variable you set in pre-requisite step and `num` parameter to `num_of_results` session variable. 
-- In new step, Store the results returned by extension in variable `extension_result`.
-- In next steps, call the `Process Result` action to do further processing on the results stored in previous step.
-- If you are planning for advanced `Filtered Search` action. Then you need to add extra step asking user the type of filter they want to apply on the result. Based on the user response each filter will be selected at runtime.
-  ![Search Action](./assets/search_action.gif)<br>
+- Open the `Search` action which should take you to an empty step 1.  Within step one, click the fx button to add a variable and add new session variable `query_text` and select "Expression" type and then put `input.original_text` as the expression. As noted in the [documentation for spell checking](https://cloud.ibm.com/docs/assistant?topic=assistant-dialog-runtime-spell-check#dialog-runtime-spell-check-how-it-works), `input.original_text` is set _only_ if the utterance from the user was altered due to spell correction and then it records the original request from the user and not the spell corrected text. Spell correction can be very counter productive for searching because it can take specialized domain vocabulary and "correct" those terms to generic words in the language, so it is often better to apply the search on the original text, as we are doing here.
+
+![Setup Search 1](./assets/search-step-1.png)<br>
+
+- Add step 2, and in step 2 change "without conditions" to "with conditions" and set the condition `query_text` is not defined. Then set the variable `query_text` to `input.text`. This is needed because the `input.original_text` is only set when spell correction changed the text. When there was no spell correction, you need to use `input.text` instead. After step 2, `query_text` is guaranteed to be the exact original query issued by the user.
+
+![Setup Search 2](./assets/search-step-2.png)<br>
+
+- Add step 3.
+  - In "Assistant says", put `Searching for: ${query_text}`.
+  - In "And then", select "Use an extension".
+    - Select the search extension.
+    - Select *Custom Search* or *Custom Search Site Restricted* as the endpoint at this step.  See the "Background" section of this document for information about the difference between the two endpoints.
+    - Set the `query` parameter to the `query_text` session variable.
+    - Set the `cx` parameter to `cx` session variable you set in pre-requisite step.
+    - Set the `num` parameter to `num_of_results` session variable.
```

Comment:
Should the wording change for [optional](https://github.com/watson-developer-cloud/assistant-toolkit/pull/60/commits/bea26e0de0b7b4f00a1e76631ef9be7b8c765567) be made here as well?

Reply:
Good point.  I have now made the change there too.